### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kriLEBED**
add greeting with my account link and my cat img.

### CODE REVIEW

**Comment by AMashoshyna** [comment]
Hi @kriLEBED ! Almost perfect =) Check link syntax:
image

**Comment by kriLEBED** [comment]
is it corect?

**Comment by AMashoshyna** [comment]
There is one more little problem: because of a space between square brackets and round brackets the link does not render as expected

**Comment by kriLEBED** [comment]
maybe this time?)

**Comment by AMashoshyna** [comment]
Ok, one last thing) Here is how you embed images:
```
![my_cat](gif/my_cat.jpg)
```

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images

BTW, in most editors you can use CTRL+SHIFT+v  to see the preview of markdown document immediately

File: README.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ It was an explosive start to front-end learning! At first it was hard for me bec
 
 ---
 
+# Hi kottans!
+My name is [Krystyna](https://github.com/kriLEBED) and my little kottan Fedir (gif/Fedir.jpg)
```
Comment (AMashoshyna) [inline README.md]:
```suggestion
My name is [Krystyna](https://github.com/kriLEBED) and my little kottan Fedir ![cat pic](gif/Fedir.jpg)
```

**Comment by AMashoshyna** [comment]
Awesome!

**Comment by kriLEBED** [comment]
Thank you for your patience))

**Pull request merged**